plugins/PrivacyManager/tests/UI/PrivacyManager_spec.js

```diff
@@ -164,6 +164,20 @@ describe("PrivacyManager", function () {
         await capturePage('gdpr_overview_no_retention');
     });
 
+    it('should load privacy settings page with config ID randomisation setting visible', async function() {
+        testEnvironment.overrideConfig('FeatureFlags', {
+          ConfigIdRandomisation_feature: 'enabled',
+        });
+        testEnvironment.save();
+
+        await loadActionPage('privacySettings');
+        await page.waitForNetworkIdle();
+        await capturePage('privacy_settings_default_with_randomisation');
+
+        delete testEnvironment.configOverride.FeatureFlags.ConfigIdRandomisation_feature;
+        testEnvironment.save();
```

Comment:
If the `capturePage` finds any difference in the screenshot this part won't be reached, creating a minor mess in the following tests.

The reset should either be done before `capturePage` (and after `page.waitForNetworkIdle`), or we wrap this in a `describe` with an additional `before` and `after`.

Reply:
I didn't think about that, good catch!